### ISSUE TITLE
Respective language titles in options

### ISSUE THREAD

**Issue by eightyy8** [recloudstream/cloudstream#339]
### Describe your suggested feature

The titles of the App language options in settings should be listed separately in the language they are in, because anyone who can't understand English will be blinded by gibberish.

Currently:
- Chinese
- English
- Spanish

But should be:
- 中文
- English
- Español

### Other details

To clarify just in case, I mean for them to be individually titled, not language specific lists of the titles

Example of what I **don't** mean:

Spanish provider
- Chino
- Inglés
- Español

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by LagradOst** [comment]
Understandable

**Comment by eightyy8** [comment]
@C10udburst Thanks! But many of them are slightly wrong, however, I've corrected them for you below to save time. Also the ordering of it shown in cs3 gui puts all lowercase languages under all the uppercase - hopefully this is fixable?

```
// https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes leave blank for auto 
 val appLanguages = arrayListOf( 
     /* begin language list */ 
     Triple("", "العربية", "ar"), 
     Triple("", "български", "bg"), 
     Triple("", "বাংলা", "bn"), 
     Triple("\uD83C\uDDE7\uD83C\uDDF7", "Brazilian Portuguese", "bp"), 
     Triple("", "čeština", "cs"), 
     Triple("", "Deutsch", "de"), 
     Triple("", "Ελληνικά", "el"), 
     Triple("", "English", "en"), 
     Triple("", "Esperanto", "eo"), 
     Triple("", "español", "es"), 
     Triple("", "فارسی", "fa"), 
     Triple("", "français", "fr"), 
     Triple("\uD83C\uDDEE\uD83C\uDDF1", "עברית", "he"), 
     Triple("", "हिन्दी", "hi"), 
     Triple("", "hrvatski", "hr"), 
     Triple("", "magyar", "hu"), 
     Triple("\uD83C\uDDEE\uD83C\uDDE9", "Bahasa Indonesia", "in"), 
     Triple("", "italiano", "it"), 
     Triple("", "ಕನ್ನಡ", "kn"), 
     Triple("", "македонски", "mk"), 
     Triple("", "മലയാളം", "ml"), 
     Triple("", "Nederlands", "nl"), 
     Triple("", "norsk nynorsk", "nn"), 
     Triple("", "norsk bokmål", "no"), 
     Triple("", "polski", "pl"), 
     Triple("\uD83C\uDDF5\uD83C\uDDF9", "português", "pt"), 
     Triple("🦍", "mmmm... monke", "qt"), 
     Triple("", "română", "ro"), 
     Triple("", "русский", "ru"), 
     Triple("", "slovenčina", "sk"), 
     Triple("", "Soomaaliga", "so"), 
     Triple("", "svenska", "sv"), 
     Triple("", "தமிழ்", "ta"), 
     Triple("", "Tagalog", "tl"), 
     Triple("", "Türkçe", "tr"), 
     Triple("", "українська", "uk"), 
     Triple("", "اردو", "ur"), 
     Triple("", "Tiếng Việt", "vi"), 
     Triple("", "中文", "zh"), 
     Triple("\uD83C\uDDF9\uD83C\uDDFC", "文言", "zh-rTW"), 
 /* end language list */ 
 ).sortedBy { it.second } //ye, we go alphabetical, so ppl don't put their lang on top
```

**Comment by C10udburst** [comment]
@eightyy8 fixed in dd38556
